results/proto/api.proto

```diff
@@ -18,6 +20,10 @@ message CreateTaskRunRequest {
 	// TODO: Accept some unique cluster identifier?
 }
 
+message DeleteTaskRunRequest {
+	string uid = 1;
```

Comment:
From https://google.aip.dev/135

```
A name field must be included. It should be called name.
- The field should be annotated as required.
- The field should identify the resource type that it references.
```

Reply:
Sure, I was using uid as name, next time we can discuss which field we use to Get, Delete or Update.